src/components/Moviepage/MoviePage.jsx

```diff
@@ -1,5 +1,33 @@
 import React from "react";
 
 export default function MoviePage({ movieInfo }) {
-  return <div>{movieInfo.id}</div>;
+  return (
+    <div>
```

Comment:
Can you think of a better html tag for a page wrapper ?

Reply:
fixed

Reply:
fixed